package.json

```diff
@@ -11,13 +11,20 @@
   },
   "scripts": {
     "update": "updates --update --minor && (git diff-files --quiet package.json || (rimraf package-lock.json node_modules && npm install))",
-    "postupdate": "git diff-files --quiet package-lock.json || (npm test -- --reporters=jest-simple-dot-reporter && npm run build)",
+    "update:python": "pipenv update --dev",
+    "postupdate": "git diff-files --quiet package-lock.json || (npm test -- --reporters=jest-simple-dot-reporter && npm run build:js)",
     "prebuild": "rimraf lib",
-    "build": "tsc -p tsconfig.json",
-    "pretest": "eslint ./",
-    "test": "jest",
+    "build:js": "tsc -p tsconfig.json",
+    "build:py": "rm -f dist/* && pipenv run python3 sync_version.py && pipenv run python3 -m build",
+    "install": "pipenv sync --dev",
+    "pretest": "pipenv run flake8 && pipenv run mypy src/grndwork_python_client && eslint src/grndwork_js_client ",
+    "publish:py": "pipenv run python3 -m twine upload dist/* && rm dist/*",
+    "publish:test": "pipenv run python3 -m twine upload --repository testpypi dist/* && rm dist/*",
+    "test": "npm run test:pytest && npm run test:jest",
+    "test:jest": "jest",
+    "test:pytest": "pipenv run pytest test/py",
     "test:watch": "nodemon -w ./ -e ts -x 'npm test -- --reporters=jest-simple-dot-reporter'",
-    "preversion": "npm test -- --reporters=jest-simple-dot-reporter && npm run build",
+    "preversion": "npm test -- --reporters=jest-simple-dot-reporter && npm run build:js",
     "postversion": "git push && git push --tags"
   },
```

Comment:
Hook into version to use node replace to update this- you can have it updated the python config. This should have the version #. Engine and app use this to do CF updates.